### PR TITLE
fix: prevent 'silent sign-in in request' error message

### PR DESCRIPTION
**Pull request by csm-thu**

### CODE REVIEW

**Review by nborde-CSM — APPROVED** [review]
LGTM :+1:

File: src/AuthMSAL/AuthMSAL.js [hunk 1]
```diff
@@ -69,6 +69,12 @@ async function _acquireTokensByRequestAndAccount(tokenReq, account) {
       if (silentTokenFetchError.errorCode === 'no_tokens_found') {
         // No token found during acquireTokenSilent, ignore this error, nothing to do
         return;
+      } else if (silentTokenFetchError.errorCode === 'login_required') {
+        console.warn(
+          'Silent authentication not possible, user is not logged in. This is usually caused when the user session ' +
```
Comment (esasova) [inline src/AuthMSAL/AuthMSAL.js:74]: Just a suggestion for phrasing: _This usually happens when_, I found _caused when_ a bit misformulated :slightly_smiling_face:

**Review by esasova — APPROVED** [review]
LGTM :+1:

File: src/AuthMSAL/AuthMSAL.js (hunk 1 above)
Comment (csm-thu) [reply src/AuthMSAL/AuthMSAL.js]: Fixed, thank you for the feedback :slightly_smiling_face:

**Pull request merged**